scripts/xtensa-build-zephyr.py

```diff
@@ -279,8 +280,26 @@ def west_init_update():
 	link back to sof/"""
 	global west_top, SOF_TOP
 	zephyr_dir = pathlib.Path(west_top, "zephyr")
-	execute_command(["git", "clone", args.url, str(zephyr_dir)], check=True, timeout=1200)
-	execute_command(["git", "fetch", "origin", args.z], check=True, timeout=300, cwd=zephyr_dir)
+
+	z_remote = "origin"
```

Comment:
Why not just add a new flag to argparser and set default value to "origin"?
I think it would make more sense as a whole.
We have a flag:
1. -u To select URL of the repo
2. -z To select branch/SHA
I would add a "-r", "--remote" flag. This would enable automation too if somebody ever needs it.

Reply:
Such a flag would be nice but it would be a different feature, it's not the purpose of this PR. The purpose of this PR is to restore the useful and previously used hack that lets anyone override the Zephyr version in ALL CIs at the same time to test Zephyr work in progress. You can't override all CIs with a new flag.

Reply:
Ok I just now understood what you meant.
You mean to change CIs behavior when a contributor pushes changes and creates PR, that makes sense now.
I was thinking at first about "internal CIs" and "why would anyone try to replace modified script in there" thus my flag proposition.